timers.js

```diff
@@ -1,4 +1,87 @@
+// addListeners: function () {
 
+  var seconds = 0
+  var timerId; //realized i don't need to set this to anything, as its just a palceholder name
 
+  var updateTime = function() {
+    seconds++
+    document.getElementById("timer").innerHTML = seconds;
+  }
 
+//StartButton is teh first time we are seeing timerId, so integer is getting stored in this variable
+var startButton = function() {
+  if (seconds === 0) {
+    document.getElementById("timer").innerHTML = 0;
+    document.body.style.background = "gray"
+  }
+  clearInterval(timerId)
+  timerId = setInterval(updateTime, 1000)
+};
 
+//had so much trouble getting pause to work. ended up being more simple than i thought. Relied on console.log to test different methods
+var pauseButton = function() {
+  clearInterval(timerId)
+  console.log("AM i paused yet?")
+  document.body.style.background = "lightgray"
+};
+
+var resetButton = function() {
+  clearInterval(timerId)
+  seconds = 0
+  document.getElementById('timer').innerHTML = "Stop Watch"
+  document.body.style.background = "black"
+};
+
+
+document.getElementById("start").addEventListener("click", startButton);
+document.getElementById("pause").addEventListener("click", pauseButton);
+document.getElementById("reset").addEventListener("click", resetButton);
```

Comment:
Overall, Excellent solution!!! Would like to see you tackle the bonus. Happy to help if you need